src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+img {
+  display: block;
+}
+
+.nav ul {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.nav__link {
+  display: block;
+  text-decoration: none;
+  color: black;
+  line-height: 60px;
+  font-weight: 500;
+}
+
+li {
+  display: inline-block;
+  text-decoration: none;
+  list-style-type: none;
+  color: black;
+  font-size: 12px;
+  font-weight: 500;
+  margin-right: 20px;
+  text-transform: uppercase;
+  font-style: normal;
+}
+
+li:last-child {
+  margin-right: 0;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.nav__link[data-qa='hover']:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  background-color: #00acdc;
```

Comment:
Create a variable for repeated color and use it everywhere

Reply:
Not fixed